keanu-project/src/main/java/io/improbable/keanu/tensor/generic/GenericTensor.java

```diff
@@ -19,14 +19,22 @@
     private long[] shape;
     private long[] stride;
 
-    public static <T> GenericTensor<T> create(T data, long[] shape) {
+    public static <T> GenericTensor<T> createFilled(T data, long[] shape) {
```

Comment:
create(T data, long[] shape) gets confused when you pass SomeType[]. It thinks T is SomeType[] instead of SomeType. You're right that if we change this here then it should be consistent with the other types. I'll rename them too?